ctapipe/image/muon/tests/test_intensity_fit.py

```diff
@@ -89,6 +89,9 @@ def test_muon_efficiency_fit(prod5_lst):
     assert u.isclose(result.impact, impact_parameter, rtol=0.05)
     assert u.isclose(result.width, ring_width, rtol=0.05)
     assert u.isclose(result.optical_efficiency, efficiency, rtol=0.05)
+    assert result.is_valid
+    assert not result.parameters_at_limit
+    assert np.isfinite(result.likelihood_value)
```

Comment:
Is is very hard to change this test so it checks that the likelihood is reasonably close to a known value?

Reply:
Testing a particular value will be hard, yes. Maybe testing that the value is lower for a fit with the correct ring parameters passed in than for one where we don't have ring center / radius values?